services/utilServices/SitesService.js

```diff
@@ -0,0 +1,133 @@
+const axios = require("axios")
+const Bluebird = require("bluebird")
+const _ = require("lodash")
+
+// Import error
+const GH_MAX_REPO_COUNT = 100
+const ISOMERPAGES_REPO_PAGE_COUNT = process.env.ISOMERPAGES_REPO_PAGE_COUNT || 3
+const ISOMER_GITHUB_ORG_NAME = process.env.GITHUB_ORG_NAME
+const ISOMER_ADMIN_REPOS = [
+  "isomercms-backend",
+  "isomercms-frontend",
+  "isomer-redirection",
+  "isomerpages-template",
+  "isomer-conversion-scripts",
+  "isomer-wysiwyg",
+  "isomer-slackbot",
+  "isomer-tooling",
+  "generate-site",
+  "travisci-scripts",
+  "recommender-train",
+  "editor",
+  "ci-test",
+  "infra",
+  "markdown-helper",
+]
+
+class SitesService {
+  constructor({ gitHubService, configYmlService }) {
+    this.githubService = gitHubService
+    this.configYmlService = configYmlService
+  }
+
+  // timeDiff tells us when a repo was last updated in terms of days (for e.g. 2 days ago,
+  // today)
+  timeDiff(lastUpdated) {
+    const gapInUpdate = new Date().getTime() - new Date(lastUpdated).getTime()
+    const numDaysAgo = Math.floor(gapInUpdate / (1000 * 60 * 60 * 24))
+    // return a message for number of days ago repo was last updated
+    switch (numDaysAgo) {
+      case 0:
+        return "Updated today"
+      case 1:
+        return "Updated 1 day ago"
+      default:
+        return `Updated ${numDaysAgo} days ago`
+    }
```

Comment:
i'm not sure that this should be done on the backend - i'm of hte opinion that the backend returns purely data and makes no judgement on how that data is consumed. this limits flexibility on the FE side (eg: if another FE component calls the same endpoint but wishes to make a `Updated last week` statement) and loses information.

Reply:
I think that makes a lot of sense, I've changed it in 66c43c8 and made an issue on the frontend so we don't forget when refactoring sites